alacritty/src/display/hint.rs

```diff
@@ -165,7 +165,7 @@ impl HintState {
 }
 
 /// Hint match which was selected by the user.
-#[derive(Clone)]
+#[derive(Debug, Clone)]
```

Comment:
Did you actually need `Debug` on this for testing, or did you just add it for different reasons?

Reply:
Yea, I found myself trying to print this out to understand things a bit better. In general, I find having `Debug` implementations is a good idea.

Why wouldn't we have them on basically everything?

Reply:
> Yea, I found myself trying to print this out to understand things a bit better. In general, I find having Debug implementations is a good idea.

Fair enough, I'm just asking if you added it out of completeness or because you actually used it. Generally my motto with `Debug` derives (at least for internal stuff) is "add only". If I ever have a reason to add it, I keep it. So if you found a reason to print this out, I have no issue with adding this. But I'm against adding things to internal stuff nobody ever had any interest in debugging.

> Why wouldn't we have them on basically everything?

Just a matter of not introducing code nobody needs.